src/Silverback.Integration/Messaging/Encryption/SymmetricEncryptionSettings.cs

```diff
@@ -59,6 +60,16 @@ public class SymmetricEncryptionSettings : EncryptionSettings
         [SuppressMessage("", "CA1819", Justification = Justifications.CanExposeByteArray)]
         public byte[]? Key { get; set; }
 
+        /// <summary>
+        ///     Gets or sets the key identifier.
+        /// </summary>
+        public string? KeyIdentifier { get; set; }
```

Comment:
I think that now we should have different settings classes for encryption and decryption, since the identifier only makes sense when encryption and the callback only when decrypting.

Reply:
I created to classes, I'm not sure about the names 😐 :
https://github.com/BEagle1984/silverback/blob/1e315aef61fcb5b5976e01b63623e3d075e96bc9/src/Silverback.Integration/Messaging/Encryption/EncryptionWithKeyRotationSettings.cs
https://github.com/BEagle1984/silverback/blob/1e315aef61fcb5b5976e01b63623e3d075e96bc9/src/Silverback.Integration/Messaging/Encryption/DecryptionWithKeyRotationSettings.cs